src/LogViewer/index.js

```diff
@@ -59,7 +61,20 @@ export default class LogViewer extends React.Component {
   }
 
   componentDidMount() {
-    this.request();
+    const { highlightStart } = this.state;
```

Comment:
`highlightStart` and the jump line should not be related for this log viewer. I know GitHub works differently, but we should introduce a new property for a `jumpToLine` in `LogViewer` so we can move to lines separately from highlighting them.